src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/CompletionTriggerAndCommitCharacters.cs

```diff
@@ -69,7 +69,13 @@ public CompletionTriggerAndCommitCharacters(LanguageServerFeatureOptions languag
         allTriggerCharacters.UnionWith(delegationTriggerCharacters);
 
         var commitCharacters = new HashSet<char>();
```

Comment:
If `languageServerFEatureOptions.UseVsCodeCompletionTriggerCharacters` isn't set, there isn't reason to create `commitCharacters`, since it'll just end up setting `AllCommitCharacters` to `[]` below.

Reply:
Agreed, pushed a commit to fix this. Thanks!